### ISSUE TITLE
generateHtmlOutput Renders encoded HTML entities

### ISSUE THREAD

**Issue by percymamedy** [advoor/nova-editor-js#79]
When using `\Advoor\NovaEditorJs\NovaEditorJs::generateHtmlOutput` it automatically encodes html entities in the returned `HtmlString` object returned:

<img width="729" alt="image" src="https://user-images.githubusercontent.com/11259669/186523638-7fb81ed5-367c-4d60-84da-cfcec586e9d7.png">
  
Then when using blade to output this string it shows as html tags which is not what we want:

![image](https://user-images.githubusercontent.com/11259669/186523877-0892d87a-c079-431f-803f-dc22a097049a.png)

I had to use this work around to make it work:

```blade
{!! html_entity_decode(\Advoor\NovaEditorJs\NovaEditorJs::generateHtmlOutput($content->body_default)) !!}
```

This does not work also:

```blade
{{ \Advoor\NovaEditorJs\NovaEditorJs::generateHtmlOutput($content->body_default) }}
```

**Comment by roelofr** [comment]
This should've been fixed by 510fea4ce8e959de29cc890ce661a5ebbbabd056. Are you using the most up-to-date views (or haven't published them)?

**Comment by percymamedy** [comment]
I am using the latest version of the package which 3.1.0
